tensorboard/webapp/runs/store/runs_selectors.ts

```diff
@@ -126,7 +126,12 @@ export const getRunSelectionMap = createSelector(
 export const getRunUserSetGroupBy = createSelector(
   getDataState,
   (dataState: RunsDataState): GroupBy | null => {
-    return dataState.userSetGroupBy ?? null;
+    return dataState.userSetGroupByKey
```

Comment:
This should be a null-check. Make sure there is a test that would have caught this. If the `userSetGroupByKey === GroupByKey.RUN`, it will hit this error (I think `GroupByKey.RUN == 0`).